.scripts/run_win_build.bat

```diff
@@ -28,8 +28,8 @@ echo Setting up configuration
 setup_conda_rc .\ ".\recipe" .\.ci_support\%CONFIG%.yaml
 if !errorlevel! neq 0 exit /b !errorlevel!
 echo Running build setup
-CALL run_conda_forge_build_setup
-
+:: Overriding global run_conda_forge_build_setup_win with local copy.
+CALL recipe\run_conda_forge_build_setup_win
```

Comment:
Note it is now calling our custom script (so we can test with it)

Reply:
Oh I thought we can't rerender after adding the local hack, so this is what I missed. Thanks, John.

Reply:
If we hacked the CI scripts directly, that is true

Though adding our own custom CI scripts like we did here, does work with re-rendering (just can be more work to setup our own script)

This logic to `conda-smithy` to specifically handling re-rendering of `conda-forge-build-setup` and then `conda-forge-ci-setup` as well as leave the door open for customization should it be needed (as is the case here)

Reply:
That's a clever trick indeed. Thanks for the tip.